Tests/API/ApiTests.cs

```diff
@@ -0,0 +1,725 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using NUnit.Framework;
+using QuantConnect.Api;
+using QuantConnect.API;
+using QuantConnect.Brokerages;
+using QuantConnect.Configuration;
+using RestSharp.Extensions.MonoHttp;
+
+namespace QuantConnect.Tests.API
+{
+    [TestFixture, Category("TravisExclude")]
```

Comment:
Should I also add the 'Ignore' attribute because these tests require configured and active Brokerage accounts?